src/static/libs/js/auth.js

```diff
@@ -42,11 +21,7 @@ var options = {
   }]
 };
 
-// TODO: setup environment variables...
-var clientId = '',
-    domain = '';
-
-var lock = new Auth0Lock(clientId, domain, options);
+window.lock = new Auth0Lock(clientId, domain, options);
```

Comment:
I removed that window variables, however window vars still exist, especially in the setWindowVars.html file.

At the moment I do not know how to pass nunjucks template variables directly into JS files.